### DISCUSSION TITLE
USER CASE DIAGRAM

### DISCUSSION THREAD

**Discussion by Aaryanb45** [Palakgupta26/Devops_VoiceRecorder#4] [General]
![Untitled](https://github.com/Palakgupta26/Devops_VoiceRecorder/assets/113382989/22645fdb-9a5a-4e45-bd37-23253aceefe0)

**Comment by Palakgupta26** [comment]
I think we have all the possible features/goals in our use case. Any further changes, we can add later.

**Comment by Aaryanb45** [comment]
ok we can have a discussion with our professor about it and afterwards we will make changes according to it.
